Great, now we can see the test which is failing in CI: 

```
======================================================================
FAIL: test_run (github.test_issue_1827.ExposureTimeCastDoubleToLong)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/exiv2/exiv2/tests/system_tests.py", line 652, in test_run
    self.compare_stderr(i, command, processed_stderr, stderr)
  File "/Users/runner/work/exiv2/exiv2/tests/system_tests.py", line 773, in compare_stderr
    self._compare_output(
  File "/Users/runner/work/exiv2/exiv2/tests/system_tests.py", line 745, in _compare_output
    self.assertMultiLineEqual(
AssertionError: "/Users/runner/work/exiv2/exiv2/src/types[207 chars]n \n" != ''
- /Users/runner/work/exiv2/exiv2/src/types.cpp:693:45: runtime error: inf is outside the range of representable values of type 'long long'
- SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /Users/runner/work/exiv2/exiv2/src/types.cpp:693:45 in 
 : Standard error does not match
 ```

It should be possible to check that test and run the exact command from a debugger to discover what is happening there.

--- a/src/easyaccess.cpp
+++ b/src/easyaccess.cpp
@@ -135,6 +135,7 @@ namespace Exiv2 {
             std::ostringstream os;
             md->write(os, &ed);
             bool ok = false;
+            if ( os.str().find("inf") != std::string::npos ) break;
             iso_val = parseInt64(os.str(), ok);
             if (ok && iso_val > 0) break;
             while (strcmp(keys[idx++], md->key().c_str()) != 0 && idx < cnt) {}

